app/views/foreman_ansible/job_templates/run_openscap_scans_-_ansible_default.erb

```diff
@@ -0,0 +1,22 @@
+<%#
+name: Run OpenSCAP scans - Ansible Default
+job_category: OpenSCAP Ansible Commands
+description_format: Run scan for all OpenSCAP policies on given hosts
+snippet: false
+provider_type: Ansible
+kind: job_template
+model: JobTemplate
+%>
+
+<% raise "Create and assign a policy to this host before proceeding" if @host.policies_enc_raw.empty? -%>
+---
+- hosts: all
+  tasks:
+<% @host.policies_enc_raw.map do |policy| -%>
+    - shell:
+        cmd: |
+        <%=       indent(10) { "/usr/bin/foreman_scap_client #{policy['id']}" %>
```

Comment:
Isn't this an overkill? Wouldn't `shell: /usr/bin/foreman_scap_client <%= policy['id'] %>` work just the same?

Reply:
I agree. I was just mimicking how the "Ansible command" job template calls the shell module. You know, since this template here is really just an Ansible command too. :-)  But I totally agree to simplify it. Will do it right away. Thanks!

Reply:
Done. Thanks again, @adamruzicka